pkg/datagatherer/versionchecker/versionchecker.go

```diff
@@ -84,6 +87,13 @@ func (c *Config) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	c.Dynamic.GroupVersionResource.Group = ""
 	c.Dynamic.GroupVersionResource.Version = "v1"
 	c.Dynamic.GroupVersionResource.Resource = "pods"
+	// node dynamic dg
+	c.DynamicNode.KubeConfigPath = aux.Dynamic.KubeConfigPath
+	c.DynamicNode.ExcludeNamespaces = make([]string, 0)
```

Comment:
Might be a more terse way to write this?

```suggestion
	c.DynamicNode.ExcludeNamespaces = []string{}
```